tutorials/install-docker-ubuntu-jammy-jellyfish/index.mdx

```diff
@@ -0,0 +1,85 @@
+---
+meta:
+  title: How to install Docker Engine on a Scaleway Instance running Ubuntu Jammy Jellyfish (22.04 LTS)
+  description: Discover how to install Docker Engine on Ubuntu Jammy Jellyfish (22.04 LTS)
+content:
+  h1: How to install Docker Engine on a Scaleway Instance running Ubuntu Jammy Jellyfish (22.04 LTS)
+  paragraph: Discover how to install Docker Engine on Ubuntu Jammy Jellyfish (22.04 LTS)
+tags: compute docker getting-started ubuntu jammy jellyfish docker-engine docker engine
+categories:
+  - compute
+dates:
+  validation: 2023-08-01
+  posted: 2023-08-01
+---
+
+Docker is a powerful and widely-used open-source platform that enables developers to automate the deployment, scaling, and management of applications in a consistent and efficient manner.
```

Comment:
I think it would be good to add a sentence clarifying Docker vs Docker Engine somewhere, e.g. "Docker Engine is Docker's core product."